.github/workflows/collector-generate-and-update.yml

```diff
@@ -23,50 +19,14 @@ jobs:
         with:
           python-version: '3.12.6'
 
-      - name: Install dependencies
```

Comment:
💬 You could have kept this step (and run also here the `apt-get`) to split the script from its dependencies

Reply:
yes, thank you I've made this change